pkg/alexa/gen/model.go

```diff
@@ -201,16 +215,19 @@ func (s *ModelSlotBuilder) WithElicitationPrompt() *ModelSlotBuilder {
 	return s
 }
 
-func (s *ModelSlotBuilder) BuildIntentSlot(locale string) alexa.ModelSlot {
-	l, _ := s.registry.Resolve(locale)
+func (s *ModelSlotBuilder) BuildIntentSlot(locale string) (alexa.ModelSlot, error) {
+	l, err := s.registry.Resolve(locale)
+	if err != nil {
+		return alexa.ModelSlot{}, err
+	}
 	ms := alexa.ModelSlot{
 		Name: s.name,
 		Type: s.typeName,
 	}
 	if s.samplesName != "" {
 		ms.Samples = l.GetAll(s.samplesName)
 	}
-	return ms
+	return ms, nil
 }
 
 func (s *ModelSlotBuilder) BuildDialogSlot(locale string) alexa.DialogIntentSlot {
```

Comment:
Why is locale passed in here. What does it do?

Reply:
Slot samples are locale based (`l.GetAll(s.samplesName)`)